test/e2e/kogitoapp_test.go

```diff
@@ -179,6 +232,34 @@ func initializeKogitoOperator(t *testing.T, f *framework.Framework, ctx *framewo
 	}
 }
 
+func initializeInfinispanOperator(t *testing.T, f *framework.Framework, ctx *framework.TestCtx) {
+	// get our namespace
+	namespace, err := ctx.GetNamespace()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Info("Installing Infinispan operator")
+
+	operatorGroup := getOperatorGroup(namespace, namespace)
+	err = f.Client.Create(goctx.TODO(), operatorGroup, &framework.CleanupOptions{TestContext: ctx, Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval})
+	if err != nil {
+		log.Fatalf("could not create operator group: %v", err)
+	}
+
+	subscription := getSubscriptionSingleNamespace("infinispan", namespace, "infinispan", "community-operators", "stable")
```

Comment:
by only creating a subscription reference were you able to deploy Infinispan on OCP 4.x? We might need that also here: https://issues.redhat.com/browse/KOGITO-703
Looking at the docs, looks like it's needed other objects to be feasible.

Reply:
I followed https://docs.openshift.com/container-platform/4.2/operators/olm-adding-operators-to-cluster.html#olm-installing-operator-from-operatorhub-using-cli_olm-adding-operators-to-a-cluster for this. According my local experiments it should be enough (tested on OCP 4.2).

Reply:
... and the `OperatorGroup` as well. I think it's safe to follow this approach in the CLI as well.
The operator appears in the `Installed Operators` tab like if it was installed via UI?

Reply:
yes, it is shown in UI, though it takes quite some time (around 2 minutes) until the operator itself is spinned up (not sure why).